src/main/java/com/ita/if103java/ims/service/impl/EventServiceImpl.java

```diff
@@ -39,7 +39,7 @@ public void create(Event event) {
         event.setDate(currentDateTime);
         event = eventDao.create(event);
         if (event.getName().isNotification()) {
-            simpMessagingTemplate.convertAndSend("/topic/event", eventDtoMapper.toDto(event));
+            simpMessagingTemplate.convertAndSend("/topic/" + event.getAccountId(), eventDtoMapper.toDto(event));
```

Comment:
Hm, @volodymyrvynnyk are you aware of this changes or this is due to not synced branches?